[4.2.0] Update instructions for Configuring IS As External IDP with OIDC (#6670)
wso2/docs-apim: en/docs/install-and-setup/setup/sso/configuring-identity-server-as-external-idp-using-oidc.md

3.  Edit the created Service Provider:

>     1.  Expand the **Claim Configuration** section. Add **http://wso2.org/claims/groups** as mandatory claim. Also add **http://wso2.org/claims/username** claim as the **Subject Claim URI**.

Reviewer: ```suggestion
    1.  Expand the **Claim Configuration** section. Add the **http://wso2.org/claims/groups** as mandatory claim. In addition, add the **http://wso2.org/claims/username** claim as the **Subject Claim URI**.
```

        </tbody>
    </table>

> ### Step - 3 Update openid OIDC scope

Reviewer: ```suggestion
### Step 3 - Update OpenID OIDC scope
```

> ### Step 3 - Update OpenID OIDC scope

Reviewer: ```suggestion
### Step 3 - Update the OpenID OIDC scope
```

> 1. Navigate to **OIDC Scopes** section under Main → Manage and list available scopes.

Reviewer: ```suggestion
1. Navigate to the **OIDC Scopes** section under **Main** → **Manage** and list available scopes.
```

> 1. Navigate to the **OIDC Scopes** section under **Main** → **Manage** and list available scopes.

Reviewer: ```suggestion
1. Navigate to the **OIDC Scopes** section under **Main** → **Manage** and list the available scopes.
```

2. Click **Add Claims** for `openid` scope, click **Add OIDC Claim**, select `groups` claim from the dropdown and click on **Add**.

>    [![add-groups-claim-to-oidc-scope]({{base_path}}/assets/img/setup-and-install/update-oidc-scope.png)]({{base_path}}/assets/img/setup-and-install/update-oidc-scope.png)

Reviewer: ```suggestion
   [![Add groups claim to the OIDC scope]({{base_path}}/assets/img/setup-and-install/update-oidc-scope.png)]({{base_path}}/assets/img/setup-and-install/update-oidc-scope.png)
```

        !!! Tip
            Instead of using the default internal roles, you can also create new roles in API Manager and map it to the provisioned users. 

>     4. Add the following claim mapping under **Claim Configuration** section:

Reviewer: ```suggestion
    4. Add the following claim mapping under the **Claim Configuration** section.
```

        <table>
        <thead>
            <tr>
                <th>Identity Provider Claim URI</th>
                <th>Local Claim URI</th>
            </tr>
        </thead>
        <tbody>
            <tr>
                <td>groups</td>
                <td>http://wso2.org/claims/role</td>
            </tr>
        </tbody>
        </table>
    
        Also select **groups** as the **Role Claim URI**.

>     [![]({{base_path}}/assets/img/setup-and-install/claim-mapping-for-sso.png)]({{base_path}}/assets/img/setup-and-install/claim-mapping-for-sso.png)

Reviewer: ```suggestion
    [![Claim mapping for sso]({{base_path}}/assets/img/setup-and-install/claim-mapping-for-sso.png)]({{base_path}}/assets/img/setup-and-install/claim-mapping-for-sso.png)
```
